docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.2"
+
+services:
+    autovoice:
+        build: .
+        image: autovoicechannels:latest
+        environment:
+            - admin_id=115271840435404809
+            - client_id=732585101400997909
+            - log_timezone=Europe/London
+            - token=NzMyNTg1MTAxNDAwOTk3OTA5.Xw2vSw.PmouyIhRtqW_cbZPy2qU1JHoLag
```

Comment:
Thats a hot token you got there

Reply:
i fixed the token issue straight away. realised as soon as i pushed but i will look into setting up the clone inside. this was more for people to run from the repo it's set more than a stand alone image which would be the ideal setup.

Reply:
just had a though. i think it would be best to not have the git clone inside the container. the dockerfile should only really be used by the pipeline and never really the end user. given the dockerfile will already sit in the repo i think it makes more sense to copy the data in than have it clone the repo from scratch.